### PR TITLE
(maint) Update submodules to pull in fixes for listing tasks/plans

### PR DESCRIPTION
**Pull request by nicklewis**
Previously, listing tasks or plans with `bolt task show` or `bolt plan
show` would fail if any task or plan was invalid. This issue was fixed
in Puppet, and this commit updates the submodule to a version that
includes the fix. This required a couple of test updates to account for
the new and improved behavior.

### CODE REVIEW

File: spec/bolt/cli_spec.rb [hunk 1]
```diff
@@ -747,12 +750,16 @@ def stub_config(config, file_content = nil)
...
-        it "task show displays an error" do
+        it "task show prints a warning but shows other valid tasks" do
           options = {
             mode: 'task',
             action: 'show'
           }
-          expect { cli.execute(options) }.to raise_error(/unexpected token at/)
+          cli.execute(options)
+          json = JSON.parse(output.string)
+          expect(json).to eq([['sample::ok', nil]])
```
Comment (zreichert) [inline spec/bolt/cli_spec.rb]: Where is the `sample::ok` task?  I don't see it in the fixtures.

Comment (nicklewis) [reply spec/bolt/cli_spec.rb]: I forgot to `git add` those! Will do.

**Comment by nicklewis** [comment]
Updated with the new fixtures included

File: spec/bolt/cli_spec.rb (hunk 1 above)
Comment (zreichert) [reply spec/bolt/cli_spec.rb]: 👍

**Pull request merged**